addon/services/cookies.js

```diff
@@ -31,6 +31,8 @@ export default Ember.Service.extend({
     return A(all).reduce((acc, cookie) => {
       if (!isEmpty(cookie)) {
         let [key, value] = cookie.split('=');
+        key   = key   || '';
```

Comment:
That's a good point. I actually don't know. I'll follow up this morning and investigate. If that's the case I'll add  a skip for that case. The bug we're running into is super weird and really hard to reproduce.